tests/e2e/framework/kubernetes.go

```diff
@@ -654,7 +654,7 @@ func (k *KubeInfo) deployIstio() error {
 
 	if *multiClusterDir != "" {
 		// Create namespace on any remote clusters
-		if err := util.CreateNamespace(k.Namespace, k.RemoteKubeConfig); err != nil {
+		if _, err := util.CreateNamespace(k.Namespace, k.RemoteKubeConfig); err != nil {
```

Comment:
In the case of multicluster testing there isn't a remote kubeclient stored so you can't make this change without storing a remote kube client somewhere

Reply:
Will look into that